### PR TITLE
[MPA] Update MPA templates for SC integration

### PR DESCRIPTION
**Pull request by harimkang**
### This PR includes :
- Rename All task template_experimental.yaml -> template.yaml
- Update template.yaml framework & hyper_parameters
- Remove 'Semi/Self' in configuration.yaml of all tasks
- Update MPA submodule: https://github.com/openvinotoolkit/model_preparation_algorithm/pull/22

### CODE REVIEW

File: external/model-preparation-algorithm/configs/rotated-detection/efficientnetb2b_maskrcnn/template.yaml [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
-model_template_id: ClassIncremental_Instance_Segmentation_MaskRCNN_EfficientNetB2B
-name: MaskRCNN-EfficientNetB2B-ClsIncr
-task_type: INSTANCE_SEGMENTATION
+model_template_id: Custom_Rotated_Detection_via_Instance_Segmentation_MaskRCNN_EfficientNetB2B
+name: MaskRCNN-EfficientNetB2B
+task_type: ROTATED_DETECTION
 task_family: VISION
 instantiation: "CLASS"
-summary: Class-Incremental Instance Segmentation for MaskRCNN-EfficientNetB2B
+summary: Class-Incremental Rotated object detection for MaskRCNN-EfficientNetB2B
```
Comment (goodsong81) [inline external/model-preparation-algorithm/configs/rotated-detection/efficientnetb2b_maskrcnn/template.yaml]: Could you add NNCF entrypoint & base_model_path?

File: external/model-preparation-algorithm/configs/rotated-detection/resnet50_maskrcnn/template.yaml [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
-model_template_id: ClassIncremental_Instance_Segmentation_MaskRCNN_ResNet50
-name: MaskRCNN-ResNet50-ClsIncr
-task_type: INSTANCE_SEGMENTATION
+model_template_id: Custom_Rotated_Detection_via_Instance_Segmentation_MaskRCNN_ResNet50
+name: MaskRCNN-ResNet50
+task_type: ROTATED_DETECTION
 task_family: VISION
 instantiation: "CLASS"
-summary: Class-Incremental Instance Segmentation for MaskRCNN-ResNet50
+summary: Class-Incremental Rotated object detection for MaskRCNN-ResNet50
```
Comment (goodsong81) [inline external/model-preparation-algorithm/configs/rotated-detection/resnet50_maskrcnn/template.yaml]: Same here.

**Review by goodsong81 — CHANGES_REQUESTED** [review]

File: external/model-preparation-algorithm/configs/rotated-detection/efficientnetb2b_maskrcnn/template.yaml (hunk 1 above)
Comment (harimkang) [reply external/model-preparation-algorithm/configs/rotated-detection/efficientnetb2b_maskrcnn/template.yaml]: I added it.

File: external/model-preparation-algorithm/configs/rotated-detection/resnet50_maskrcnn/template.yaml (hunk 2 above)
Comment (harimkang) [reply external/model-preparation-algorithm/configs/rotated-detection/resnet50_maskrcnn/template.yaml]: I added it.

**Comment by harimkang** [comment]
run ote-test

**Comment by harimkang** [comment]
MPA API tests: https://ci-ote.iotg.sclab.intel.com/job/ote/job/pr-model-preparation-algorithm-test/40/

**Review by goodsong81 — APPROVED** [review]
LGTM.

**Review by chuneuny-emily — APPROVED** [review]

**Pull request merged**